pkg/jobrunaggregator/jobrunaggregatorapi/types_row_historical_data.go

```diff
@@ -19,6 +19,8 @@ type HistoricalJobData struct {
 	Architecture string
 	Network      string
 	Topology     string
+	// JobRuns is the number of job runs that were included when we queried the historical data.
+	JobRuns int
```

Comment:
This is a cool piece of information, do you think it's helpful to present it in the resulting template table?

Reply:
Yes it is, because if the different backends for the same nurps do not have the same count, we've got a problem. That was what exposed we were not properly recording zeros. Hopefully they'll all be synced up by Friday. I'll get this into the table.

Reply:
This is added, it required displaying it for alerts and disappearing job tables as well, but I will deal with alerts in the future.

Reply:
Yeah, I should rethink how that table is being generated. This is great!